src/SIL.LCModel/DomainImpl/ReadWriteServices.cs

```diff
@@ -128,30 +130,42 @@ internal static Guid LoadGuid(XElement reader)
 		/// </summary>
 		internal static DateTime LoadDateTime(XElement reader)
 		{
-			if (reader == null) throw new ArgumentNullException("reader");
+			if (reader == null) throw new ArgumentNullException(nameof(reader));
 
-			// ENHANCE: it would be better to use DateTime.Parse instead of parsing this ourselves.
```

Comment:
can't use DateTime.Parse with the European date format dd.mm.yyyy (not sure if any projects have this problem)